app/src/controllers/project/sagas.js

```diff
@@ -255,12 +255,19 @@ function* watchUpdatePAState() {
 }
 
 function* fetchProject({ payload: { projectId, isAdminAccess } }) {
-  const project = yield call(fetch, URLS.project(projectId));
-  yield put(fetchProjectSuccessAction(project));
-  yield put(setProjectIntegrationsAction(project.integrations));
-  if (!isAdminAccess) {
+  let project = {};
+  if (isAdminAccess) {
+    try {
+      project = yield call(fetch, URLS.project(projectId));
+    } catch (error) {
+      yield put(showDefaultErrorNotification(error));
+    }
+  } else {
+    project = yield call(fetch, URLS.project(projectId));
```

Comment:
What's the difference with line 261?

Reply:
I used to think that would be great to catch error on get project only for admin access, but for now I think that it isn't bad idea to handle error on every request.
Fixed.